pkg/openvpn/assets/client-config.template

```diff
@@ -8,6 +8,11 @@ pull
 data-ciphers AES-256-GCM
 tls-client
 
+link-mtu 1440
+mssfix 1340
```

Comment:
In the [OpenVPN Reference](https://openvpn.net/community-resources/reference-manual-for-openvpn-2-6/) for `--mssfix` I read

> The `--mssfix` option only makes sense when you are using the UDP protocol for OpenVPN peer-to-peer communication, i.e. `--proto udp`

As we are always using TCP, it's not clear to me what difference this parameter can make.

Reply:
I've seen a sporadic issue, that seemed to have disappeared after setting `--mssfix`. 
Though, now I have seen the issue again and turned down the MTU to the MTU that was cached at a route:
```
/# ip r get 100.80.0.10
100.80.0.10 via 10.181.18.223 dev tunl0 src 100.80.2.1 uid 0
    cache expires 449sec mtu 1312
```